modules/fedimint-wallet-tests/tests/tests.rs

```diff
@@ -27,7 +27,7 @@ fn bsats(satoshi: u64) -> bitcoin::Amount {
 
 const PEG_IN_AMOUNT_SATS: u64 = 5000;
 const PEG_OUT_AMOUNT_SATS: u64 = 1000;
-const PEG_IN_TIMEOUT: Duration = Duration::from_secs(30);
+const PEG_IN_TIMEOUT: Duration = Duration::from_secs(60);
```

Comment:
Nope. :D . But unless we ever expect to wait for all this time anywhere, I guess it doesn't hurt to have these be a bit longer, in case CI machine is extra slow.